op-test

```diff
@@ -692,3 +692,19 @@ except Exception as e:
     optestlog.error("Exit unexpectedly with Exception={}".format(e))
     OpTestConfiguration.conf.util.cleanup()
     exit(-1)
+finally:
+    # Create a softlink to `latest` test results
+    output = OpTestConfiguration.conf.logdir
+    if not os.path.exists(output):
+        optestlog.error("Test result folder not found")
+        sys.exit(-1)
+    basedir = os.path.abspath(os.path.join(output, os.pardir))
+    latest = os.path.join(basedir, "latest")
+    if os.path.exists(latest):
+        if os.path.islink(latest):
+            os.unlink(latest)
+    try:
+        os.symlink(output, latest)
+    except OSError:
+        optestlog.error('Unable to create test result latest softlink')
+    sys.exit(0)
```

Comment:
If I'm reading this corretly, it'll have a 0 exit code even if tests fail. This would prevent op-test from being used in a variety of scripting environments.

I think it's important to maintain the exit code being about if the tests succeeded or not, as I use this functionality to script things.

Reply:
@sammj @stewart-ibm sure, got, my bad, will retain the exit status from the try block...

Reply:
Almost looks good, but here the finally block changes the exit code: https://github.com/open-power/op-test-framework/pull/327/commits/3398d6f6ee868af1a03ff32a34108815156d2355#diff-bb3c3a974ddd71aa44e038ac6af308dcR744
I think the finally block shouldn't change the exit code at all; failing to create the symlink is annoying but probably shouldn't "fail" a test that actually succeeded. Thoughts @sathnaga @stewart-ibm ?

Reply:
@sammj here is only place we are changing the return code in finally that too if could not find the results folder, not for symlink creation failure 
```
    if not os.path.exists(output):
        optestlog.error("Test result folder not found")
        exit_code = -1
        sys.exit(exit_code)
```

correct me if I missed anything?

Reply:
Yep, but that would still override the result of the test right?

Reply:
yes, and having the test folder missing is a issue to be reported even if the testcase pass right?,
I doubt this case would not hit in most(99.9%) of the scenarios, just that (.1%) handle...

Reply:
I agree that it should be reported, but the exit code represents whether the _test_ succeeded or not. I think it would be better to print a big warning, but maintain the exit code from the test.

Reply:
@sammj Sure, we can do it...I will change it.

Reply:
@sammj have updated nw.